src/Tizen.NUI.Components/Controls/TabView.cs

```diff
@@ -252,6 +252,38 @@ public void RemoveTab(int index)
             }
         }
 
+        /// <summary>
+        /// Adds a tab from the given TabItem.
+        /// TabItem contains Title and IconURL of a new TabButton in TabBar and Content of a new View in TabContent.
+        /// <param name="tabItem">The tab item which contains Title and IconURL of a new TabButton in TabBar and Content of a new View in TabContent.</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void Add(TabItem tabItem)
+        {
+            var hasTitle = (String.IsNullOrEmpty(tabItem.Title) == false);
+            var hasIconURL = (String.IsNullOrEmpty(tabItem.IconURL) == false);
+
+            // Adds a new TabButton and Content View only if TabItem has TabButton properties and Content View.
+            if ((hasTitle || hasIconURL) && (tabItem.Content != null))
+            {
+                var tabButton = new TabButton(tabItem.TabButtonStyle);
```

Comment:
Please suppress the warning complaining about disposing `TabButton`.

Reply:
Thank you for your comment!
I have added the SuppressMessage.